cypress/support/pages/product.pageObject.js

```diff
@@ -0,0 +1,15 @@
+import PageObject from '../PageObject';
+
+class ProductPageObject extends PageObject {
+  clickAddToCart() {
+    cy.contains('.btn', 'Add to cart').click();
+  }
+
+  allertMessage(message) {
```

Comment:
The function name 'allertMessage' seems to be a typo. Please correct it to 'alertMessage'.

Reply:
Done